bndtools.core.test/src/simple/package-info.java

```diff
@@ -0,0 +1,3 @@
+@org.osgi.annotation.bundle.Export
```

Comment:
simple.bnd already has these packages as exported, so we can remove the duplicate Export annotation here.